lib/api/corsPreflight.js

```diff
@@ -76,7 +76,8 @@ export default function corsPreflight(request, log, callback) {
 
         const resHeaders = gatherCorsResHeaders(corsRule, corsOrigin,
             corsMethod, corsHeaders);
-        // TODO: pushMetric('corsPreflight', log, { bucket: bucketName });
+        // TODO: add some level of metrics for non-standard API request:
```

Comment:
this and the serve/head website methods are kind of non-standard because they do not require account credentials; Rahul wants to determine whether Utapi should use a different schema for them before we push metrics. We started an issue to track them here:
https://github.com/scality/utapi/issues/88